tests/testthat/test-transition_risk_score.R

```diff
@@ -0,0 +1,22 @@
+test_that("`transition_risk_score` and `benchmark_tr_score` has NA due to
+          NA in either column `profile_ranking` or `reduction_targets`", {
```

Comment:
The only test covers a very narrow edge case. Consider testing:

1. Simplest correct usage.
2. More edge cases.
3. Errors, e.g. if crucial columns are missing.
4. Conceptual cases (with the help of Anne/Tilman?).